lib/beacon_api/controllers/v1/beacon_controller.ex

```diff
@@ -0,0 +1,11 @@
+defmodule BeaconApi.V1.BeaconController do
+  use BeaconApi, :controller
+
+  @spec get_state_root(Plug.Conn.t(), any) :: Plug.Conn.t()
```

Comment:
we should handle 400, 404, etc.. for this scenario

Reply:
If we add this in this PR, we may take the following strategy:
1. If asking for `state_id in ["head", "genesis", "finalized", "justified"]` then return the hardcoded block root.
3. If asking for the hardcoded block using the root, return the hardcoded block root.
4. If asking for any other block using a root or a number, return a 404.
5. If state_id is not a number, not in block root format (0x and 32 bytes) or one of the checkpoints/head/genesis, then return 400.

Regarding 5, I think it may be easier to implement it outside of this PR, after integrating with OpenApiSpex (I'm adding an issue for that right now).

Reply:
What is 5 in "Regarding 5" ?